javaparser-core/src/main/javacc/java_1_7.old

```diff
@@ -1,20 +1,19 @@
 /*
```

Comment:
@SmiddyPence kept it when implementing #27. It was quite useful to hunt down & understand the lambda stuff tbh. I'd keep it until we add Java 9 support :)